htdocs/js/jquery.dynamictable.js

```diff
@@ -72,21 +101,58 @@
         }
 
 
-    };
+    }, wrapTable = function (table) {
+        $(table).wrap("<div class=\"row\"></div>");
+        // Add wrapper code necessary for bootstrap carousel
+        $(table).wrap("<div class=\"carousel slide col-xs-12\"></div>");
 
-    $.fn.DynamicTable = function () {
-        this.filter("table").each(function () {
-            var leftLink, rightLink, table = this;
-            // Add wrapper code necessary for bootstrap carousel
-            $(this).wrap("<div class=\"carousel slide\" data-ride=\"carousel\"></div>");
-            $(this).wrap("<div class=\"carousel-inner\"></div>");
+        // Add wrapper necessary for dynamictable code
+        $(table).wrap("<div class=\"dynamicContentWrapper table-scroll\" style=\"overflow-x: auto\"></div>");
 
-            // Add wrapper necessary for dynamictable code
-            $(this).wrap("<div class=\"dynamicContentWrapper table-scroll\" style=\"overflow-x: auto\"></div>");
+        // Add links for carousel
+        $(table).after('<a class="left carousel-control" href="#"><span class="glyphicon glyphicon-chevron-left"></span></a><a class="right carousel-control" href="#" data-slide="next"><span class="glyphicon glyphicon-chevron-right"></span></a>');
```

Comment:
I think I finally understand why you're using $(".left") below.. maybe this class should be called something like "leftLink" or "leftCarousel" to make the javascript easier to follow.